docs/user/tutorial/60-admin.rst

```diff
@@ -112,19 +113,16 @@ Let's move a file.
        :alt: the 'Cut' icon
        :width: 48
 
-#.  Move it to the clipboard, using |cut|.
+#.  Move it to the clipboard, using the |cut| icon.
```

Comment:
``|cut|`` was correct here, because the alt text already says ``the 'Cut' icon`` - so that would become ``using the the 'Cut' icon icon``